Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectBuilder.cs

```diff
@@ -27,8 +27,10 @@ public enum ProgressMessageType
         public const string RebuildTargetArgument = "-t:Rebuild";
         public const string DefaultBuildArguments = MSBuildProjectBuilder.BuildTargetArgument;
 
+        private const string adoAuthenticationUrl = "https://microsoft.com/devicelogin";
```

Comment:
>adoAuthenticationUrl [](start = 29, length = 20)

NIT - I haven't noticed the usage of lowercase to indicate static const before :/ Personally I prefer ALL_CAPS_WITH_UNDERSCORES to denote constants. Feel free to disregard.

Reply:
That's so C++ of you @coprez. =) The [.NET Guidelines on identifier capitalization](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers) is to use PascalCase for everything except parameters.

Reply:
The [field name guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields) reinforce this as well.

Reply:
> _The field-naming guidelines apply to static public and protected fields. Internal and private fields are not covered by guidelines_
But I will change it if that's what people prefer.